docs/source/support/releases.rst

```diff
@@ -0,0 +1,86 @@
+
+.. _release-notes:
+
+Release Notes
+-------------
+
+Release notes for each release are published on the `Release Notes section of the DAML Driven blog <https://daml.com/release-notes/>`_.
+
+.. _roadmap:
+
+Roadmap
+-------
+
+Once a month Digital Asset publishes a community update to accompany the announcement of the release candidate for the next release. The community update contains a section outlining the next priorities for development. You can find community updates on the `DAML Driven Blog <https://daml.com/daml-driven>`_, or subscribe to the mailing list or social media profiles on `https://daml.com/ <https://daml.com/>`_ to stay up to date.
+
+.. _release_process:
+
+Process
+-------
+
+Weekly snapshot and monthly stable releases follow a regular process and schedule. The process is documented `in the DAML repository <https://github.com/digital-asset/daml/blob/master/release/RELEASE.md>`_ so only the schedule for monthly releases is covered here.
+
+Selecting a Release Candidate
+
+  This is done by the DAML Core engineering teams on the **first Monday of every month**.
+
+  The monthly releases are time-based, not scope-based. Furthermore, DAML development is fully HEAD-based so both the repository and every snapshot are intended to be in a fully releasable state at every point. The release process therefore starts with "selecting a Release Candidate". Typically the Snapshot from the preceding Wednesday is selected as the Release Candidate.
+
+Release Note and Candidate Review
+
+  After selecting the Release Candidate, Release Notes are written and reviewed with a particular view towards unintended changes and violations of :ref:`Semantic Versioning <versioning>`.
+
+Release Candidate Refinement
+
+  If issues surface in the initial review, the issues are resolved and different Snapshot is selected as the Release Candidate.
+
+Release Candidate Announcement
+
+  Barring delays due to issues during initial review, the Release Candidate is announced publicly with accompanying  Release Notes on **the Thursday following the first Monday of every Month**.
+
+Communications, Testing and Feedback
+
+  In the days following the announcement, the release is presented and discussed with both commercial and community users. It is also put through its paces by integrating it in `project:DABL <https://projectdabl.com>`_ and several ledger integrations. 
+
+Release Candidate Refinement II
+
+  Depending on feedback and test results, new Release Candidates may be issued iteratively. Depending on the severity of changes from Release Candidate to Release Candidate, the testing period is extended more or less.
+
+Release
+
+  Assuming the release is not postponed due to extended test periods or newly discovered issues in the Release Candidate, the release is declared stable and given a regular version number on **the second Wednesday after the first Monday of the Month**.
+
+.. figure:: release-timeline.png
```

Comment:
Since there is no way to comment directly on `release-timeline.png`: 
Is this Wednesday/Tuesday disparity on purpose?
![image](https://user-images.githubusercontent.com/29121423/94111979-0fd46e00-fe45-11ea-97a0-82fa2251fc0d.png)

Reply:
Good spot, thank you.